### ISSUE TITLE
Just for inference or need training?

### ISSUE THREAD

**Issue by Wand-LostWitch** [Inception3D/TTT3R#17]
Hi! Thanks for your amazing work. I'm not sure whether I'd got you. Does your method use the weights trained by cut3r and just use different rule of updating the learning rate at inference time? Or you retrain the model with the new way of updating the learning rate \beta？

**Comment by rover-xingyu** [comment]
Hi @Wand-LostWitch, thanks for your interest! 

You are correct, our test-time training learning rate is derived from the closed-form solution of cross-attention guided confidence. Therefore, you can directly reuse the weights of CUT3R without any need for fine-tuning.
